examples/crowdfund.vy

```diff
@@ -56,6 +56,6 @@ def refund():
             return
 
         send(self.funders[i].sender, self.funders[i].value)
-        clear(self.funders[i])
+        empty(self.funders[i])
```

Comment:
I think this would have to be `self.funders[i] = empty(Funder)`

Reply:
oic, in this branch you have changed the `clear` function (which zeroes its argument) to `empty`

Reply:
Oh, we actually didn't want to do that, did we? Been so long I forgot, but it should either be `self.funders[i] = empty(Funder)` or `clear(self.funders[i])` because `empty(self.funders[i])` doesn't make sense.

Am I rehashing the past again?